### ISSUE TITLE
Upload tags

### ISSUE THREAD

**Issue by dauris** [websemantics/vimeo-upload#32]
Is it possible to upload the video with tags along with the title and description

**Comment by ivanhaentjens** [comment]
yes, could you please explain? I need to implement this feature too. Thanks in advance.
